bazel/repository_locations.bzl

```diff
@@ -5,7 +5,7 @@
 REPOSITORY_LOCATIONS = dict(
     boringssl = dict(
         # Use commits from branch "chromium-stable-with-bazel"
-        commit = "b4061d947ac3aee79235a7ece0f500d8005c80ff",  # chromium-65.0.3325.146
+        commit = "9df0c47bc034d60d73d216cd0e090707b3fbea58",  # chromium-65.0.3325.146
```

Comment:
Why does SHA change but not version? Also, as long as I'm typing, is there any reason that bazel is not merged to the normal stable branch?

Reply:
Because I was looking at the wrong place in Chromium source tree :( It's synced often enough that this is the first time it was out of sync, but ultimately there were few commits missing from what was shipped in 65.0.3325.146.

wrt branch, BoringSSL team updates it manually the week after Chrome ships, so I expect this is going to be fixed by next week.